src/bin/eolian/main.c

```diff
@@ -516,9 +517,79 @@ init_cmdline_options(CmdLineOptions* options) {
     };
 }
 
+int
+process_options(int argc, char** argv, CmdLineOptions* options);
+
+void
+read_options_from_response_file(CmdLineOptions* options, const char* filename)
+{
+    printf("Reading eolian gen options from %s\n", filename);
+    fflush(stdout);
+
+    // Reads the whole content into memory
+    FILE *f = fopen(filename, "r");
+
+    printf(":: File opened\n"); fflush(stdout);
+
+    fseek(f, 0, SEEK_END);
+    long file_length = ftell(f);
+    fseek(f, 0, SEEK_SET);
+
+    printf("::   File size: %ld\n", file_length); fflush(stdout);
+
+    char *contents = malloc(sizeof(char) * (file_length + 1));
+
+    printf(":: Starting to read contents...\n"); fflush(stdout);
+    fread(contents, sizeof(char), file_length, f);
+    contents[file_length] = '\0';
+    printf("::   * Done. (bust: %s)\n", &contents[file_length - 10]); fflush(stdout);
+
+    // No need for the file anymore
+    fclose(f);
+    printf(":: File Closed.\n"); fflush(stdout);
+
+    // Now we finally parse arguments
+
+    // Count how many arguments we have
+    printf(":: Counting args...\n"); fflush(stdout);
+    int argc = 0;
+    for (int i = 0; i < file_length; ++i) {
+        switch (contents[i])
+        {
+            case '\n':
+                ++argc;
+            case '\r':
+                contents[i] = '\0';
+            default:
+                break;
+        }
+    }
+    printf("::   Total arguments: %d\n", argc); fflush(stdout);
```

Comment:
Is there a need for a flush in every print or is it for debugging?

Reply:
Debugging :)